read-fonts/build.rs

```diff
@@ -8,23 +8,26 @@
 
 use std::path::Path;
 
-static FILES_TO_MOVE: &[&str] = &[
-    "../resources/test_fonts/ttf/cmap14_font1.ttf",
-    "../resources/test_fonts/ttf/linear_gradient_rect_colr_1.ttf",
-    "../resources/test_fonts/ttf/simple_glyf.ttf",
-    "../resources/test_fonts/ttf/vazirmatn_var_trimmed.ttf",
-    "../resources/test_fonts/extracted/vazirmatn_var_trimmed-glyphs.txt",
-];
-
 fn main() {
+    println!("cargo:rerun-if-changed=../resources/test_fonts/");
```

Comment:
you may need to add `read-fonts/build.rs` to the `resources/scripts/println_ignore.txt` allowlist, otherwise after #313 was merged the prepush check for println! would trigger at this line (merge or rebase `main` to verify)

Reply:
Thanks for catching this! I didn't realize that PR had been merged. Fixed.